tests/CMakeLists.txt

```diff
@@ -11,6 +11,30 @@ target_include_directories(utils INTERFACE
     .
 )
 
+function(cuke_set_environment environment)
+    set(options)
+    set(oneValueArgs )
+    set(multiValueArgs RUNPATH)
+    
+    cmake_parse_arguments(CUKE_ENV "${options}" "${oneValueArgs}" "${multiValueArgs}" ${ARGN})
+    
+    if(NOT "${CUKE_ENV_UNPARSED_ARGUMENTS}" STREQUAL "")
+        message(
+            FATAL_ERROR
+                "unparsed arguments in call to cuke_set_environment: ${CUKE_ENV_UNPARSED_ARGUMENTS} from ${CMAKE_CURRENT_LIST_FILE}"
+        )
+    endif()
+
+    if( NOT "${CUKE_ENV_RUNPATH}" STREQUAL "")
+        if(WIN32)
+            string(REPLACE ";" "\;" CUKE_ENV_RUNPATH "${CUKE_ENV_RUNPATH}")
+        endif()
+        set(RUNPATH "$<IF:$<PLATFORM_ID:Windows>,PATH,LD_LIBRARY_PATH>")
+        list(APPEND environment "$<IF:$<PLATFORM_ID:Windows>,PATH,LD_LIBRARY_PATH>=path_list_prepend:$<SHELL_PATH:${CUKE_ENV_RUNPATH}>")
+    endif()
+    set(${environment} ${${environment}} PARENT_SCOPE)
+endfunction()
+
```

Comment:
This is a lot of code to just support Windows. Is it possible to move this to run-windows.ps1 (if it makes sense).

@cwellm any ideas from your side?